### PR TITLE
Drop delete col in action_runner_token and action_runner table

### PR DESCRIPTION
**Pull request by nanguanlin6**
I accidentally ran into this because the act runner cannot register now due to this field.

It's a bug caused by xorm v1.3.3(https://gitea.com/xorm/xorm/issues/2339). But I think dropping those fields is better since they are not used.

### CODE REVIEW

**Comment by JakobDev** [comment]
You can drop both in one Migration. No need to create 2.

**Comment by lng2020** [comment]
> You can drop both in one Migration. No need to create 2.

Thanks. Updated.

**Comment by lunny** [comment]
https://gitea.com/xorm/xorm/pulls/2341 merged.

File: models/migrations/v1_22/v282.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"xorm.io/xorm"
+)
+
+func DropDeletedCol(x *xorm.Engine) error {
+	sess := x.NewSession()
+	defer sess.Close()
+	if err := sess.Begin(); err != nil {
+		return err
+	}
+	if err := base.DropTableColumns(sess, "action_runner_token", "deleted"); err != nil {
```
Comment (lunny) [inline models/migrations/v1_22/v282.go:18]: Then all deleted records will become undeleted.

Comment (nanguanlin6) [reply models/migrations/v1_22/v282.go:18]: For action_runner that's true. Sorry for missing that.
For action_runner_token, we did not delete them. But use the `is_active` field to activate/deactivate them.

**Comment by lng2020** [comment]
Never mind. Probably it's an unnecessary change. (Though it's a little weird that the two tables are the only tables in the whole codebase using the `deleted` field)

**Pull request closed without merging**